doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -911,6 +917,34 @@ mylibvar = Libdl.dlopen("mylib")
 ccall(@dlsym("myfunc", mylibvar), Cvoid, ())
 ```
 
+## Closure cfunctions
+
+The first argument to [`@cfunction`](@ref) can be marked with a `$`, in which case
+the return value will instead be a `struct CFunction` which closes over the argument.
```

Comment:
Out of curiosity: what happens if I don't use a `$`, but I give a global name that's bound to a non-singleton function object?

Reply:
Like `ccall`, it's resolved by the macro to close over the value (not the binding), since that's almost always what you want anyways, and makes it easier to avoid accidentally shooting your self in the (gc) foot.